{"uuid":"7e17c522-8c55-45dc-9cf9-758b7eafdfd5","historyId":"cb5af1b873d20ae28a66748f97f6f4cb","fullName":"org.example.HelloWorld_Selenium.Hello_test","labels":[{"name":"package","value":"org.example.HelloWorld_Selenium"},{"name":"testClass","value":"org.example.HelloWorld_Selenium"},{"name":"testMethod","value":"Hello_test"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"Selenium_Project"},{"name":"subSuite","value":"org.example.HelloWorld_Selenium"},{"name":"host","value":"DESKTOP-DBHECHF"},{"name":"thread","value":"16832@DESKTOP-DBHECHF.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Hello_test","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.net.MalformedURLException: no protocol: https//:kavyas_OjNri8:PDnTspzUfBetqyssJEMA@hub-cloud.browserstack.com/wd/hub","trace":"java.lang.RuntimeException: java.net.MalformedURLException: no protocol: https//:kavyas_OjNri8:PDnTspzUfBetqyssJEMA@hub-cloud.browserstack.com/wd/hub\r\n\tat org.example.HelloWorld_Selenium.Hello_test(HelloWorld_Selenium.java:36)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:578)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:848)\r\n\tat org.testng.TestRunner.run(TestRunner.java:621)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\r\n\tat org.testng.TestNG.run(TestNG.java:1082)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\nCaused by: java.net.MalformedURLException: no protocol: https//:kavyas_OjNri8:PDnTspzUfBetqyssJEMA@hub-cloud.browserstack.com/wd/hub\r\n\tat java.base/java.net.URL.<init>(URL.java:764)\r\n\tat java.base/java.net.URL.<init>(URL.java:646)\r\n\tat java.base/java.net.URL.<init>(URL.java:582)\r\n\tat org.example.HelloWorld_Selenium.Hello_test(HelloWorld_Selenium.java:34)\r\n\t... 25 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1689250978018,"stop":1689250978046}